jkube-kit/common-maven/src/test/java/org/eclipse/jkube/kit/common/util/MavenUtilTest.java

```diff
@@ -131,13 +128,10 @@ public void testGetTransitiveDependencies(@Mocked MavenProject mavenProject) {
     // When
     final List<Dependency> result = MavenUtil.getTransitiveDependencies(mavenProject);
     // Then
-    assertThat(result, hasSize(2));
-    assertThat(result, contains(
-        equalTo(Dependency.builder().groupId("org.eclipse.jkube").artifactId("foo-dependency").version("1.33.7")
-            .type("jar").scope("runtime").build()),
-        equalTo(Dependency.builder().groupId("org.eclipse.jkube").artifactId("bar-dependency").version("1.33.7")
-            .type("jar").scope("runtime").build())
-    ));
+      assertThat(result).hasSize(2);
```

Comment:
Join these multiple assertions subject to one assertion chain.

```
assertThat(result)
  .hasSize(2)
  .contains(Dependency.builder().groupId("org.eclipse.jkube").artifactId("foo-dependency").version("1.33.7")
              .type("jar").scope("runtime").build(), Dependency.builder().groupId("org.eclipse.jkube").artifactId("bar-dependency").version("1.33.7")
              .type("jar").scope("runtime").build());
```

Reply:
Allow me to change according to suggested comments.